show_client/show_opts.go

```diff
@@ -0,0 +1,66 @@
+package show_client
+
+import (
+	sdc "github.com/sonic-net/sonic-gnmi/sonic_data_client"
+)
+
+const (
+	SHOW_CMD_OPT_UNIMPLEMENTED_DESC = "UNIMPLEMENTED"
+	SHOW_CMD_OPT_DISPLAY_DESC       = "[display=all] Show internal interfaces [default: all]"
+	SHOW_CMD_OPT_VERBOSE_DESC       = "[verbose=true] Enable verbose output"
+	SHOW_CMD_OPT_INTERFACES_DESC    = "[interfaces=TEXT] Filter by interfaces name"
+	SHOW_CMD_OPT_INTERFACE_DESC     = "[interface=TEXT] Required interface name"
+	SHOW_CMD_OPT_PERIOD_DESC        = "[period=INTEGER] Display statistics over a specified period (in seconds)"
+	SHOW_CMD_OPT_JSON_DESC          = "[json=true] No-op since response is in json format"
+)
+
+var (
+	SHOW_CMD_OPT_VERBOSE = sdc.NewShowCmdOption(
+		"verbose",
+		sdc.Optional,
+		SHOW_CMD_OPT_VERBOSE_DESC,
+		sdc.BoolValue,
+	)
+
+	SHOW_CMD_OPT_NAMESPACE = sdc.NewShowCmdOption(
+		"namespace",
+		sdc.Unimplemented,
+		SHOW_CMD_OPT_UNIMPLEMENTED_DESC,
+		sdc.StringValue,
+	)
+
```

Comment:
For some transceiver command, we also have an option `-hw, --fetch-from-hardware`, which is also Unimplemented. Would you mind adding this option, or we can add it later in another PR.